chapters/Chapter 1 — Untitled.md

```diff
@@ -87,38 +86,46 @@ Leanne took an explosive from her belt, and planted it on the hull, just behind
+
+The cards was small. There was a little yellow insignia on the upper-left of a diving eagle: the Leesomes, if Leanne remembered correctly. They were arrangers. There was little information on the card, besides a picture of her and her ship, and a location she once frequented years ago. It was extremely impressive that her attackers had found her at all.
+
+There was another card. Same insignia. It was odd. Although arrangers would give the same card to as many assassins as would take them, they would rarely give one assassin more than one card at once. Leanne always assumed it because the cards themselves were expensive due to whatever cryptographic chips were on them, and that the arrangers didn't want the assassins hoarding the cards hoping they'd get lucky and kill one of the targets eventually.
 
-She checked the feeds from the cameras. It appeared she was alone, or alone enough. She rifled through the console. Her attackers, like her, were assassins. They should have a card for the job. It would have all the pertinent details, and they’d turn it in with proof of death. 
+On the second card was a woman. She lived on Dressara. Short black hair. Eyes aglow. Chin up. Her lips—outlined in dark red—halfway between a smile and a smirk. And, according to the card, not all that dangerous.
 
-It was unlikely there’d be anything on the card to trace back to the submitter; assassins always worked through an Arranger, and even the Arranger rarely knew who hired them. But Leanne searched all the same, whether out of habit or hunch.
+Could be important. Perhaps there was a link between Leanne and this woman. Or perhaps she was just another job, handed out at the same time. There were two attackers, after all; perhaps the Leesomes had given them each their own job.
 
-The card was small. There was a little yellow insignia on the upper-left of a diving eagle—the Leesome Arrangers, if Leanne remembered correctly. There was little information on it, besides a picture of her and her ship, and a location she once frequented years ago. It was extremely impressive that they found her at all. 
+Leanne pocketed the card. She looked out the window. Her ship was a speck in the distance. 
 
-There was another card. Same insignia. It was odd: Arrangers usually only handed out one card at a time to any given assassin. Leanne always assumed it was because the cards themselves were expensive due to their cryptographic something or others. 
+She glanced around the remains of the cockpit. It looked like it could still be flyable. 
 
-On the second card was a woman. She lived on Dressara. Short black hair, angular face, and according to the card, not all that dangerous.
+She flicked the switches. Pulled the handles. The ship’s many engines weren’t meant to be controlled by a human, but she was managing to bring it around gracefully enough.
```

Comment:
maaaaybe mention that they're meant to be controlled by alts, again, but maybe not: don't want to be too repetitive.